en/peripherals/companion_computer_peripherals.md

```diff
@@ -44,10 +44,31 @@ Cameras are among the most common form of data collection on a drone whether it
 
 ### Stereo Cameras
 
-Stereo cameras are typically used for depth perception, path planning and SLAM. They are in no way guaranteed to be plug and play with your companion computer.
+Stereo cameras are typically used for depth perception, path planning and SLAM. 
+They are in no way guaranteed to be plug and play with your companion computer.
 
 #### Where To Buy
 * [Intel® RealSense™ Depth Camera D435](https://click.intel.com/intelr-realsensetm-depth-camera-d435.html)
 * [Intel® RealSense™ Depth Camera D415](https://click.intel.com/intelr-realsensetm-depth-camera-d415.html)
 * [DUO MLX ](https://duo3d.com/product/duo-minilx-lv1)
 * [Ironsides](https://www.perceptin.io/products)
+
+
+## Data Telephony (LTE) {#data_telephony}
+
+An LTE USB module can be attached to a companion computer and used to route MAVLink traffic between the flight controller and the Internet.
+
+There is no "standard method" for a ground station and companion to connect over the Internet (i.e. for the GCS to discover the vehicle IP address, and visa versa).
+Depending on your mobile network, it is likely that you won't be able to just directly connect to the remote QGC computer IP over the Internet.
+
+A common approach is to set up a virtual private network between the companion and GCS computer (i.e. install a VPN system on both computers).
```

Comment:
-Are these two new paragraphs correct/OK?
- Is there a vpn system/client we recommend or have used that I could use here as an example?

Reply:
I would not suggest the broadcasting by default, as it can create more traffic than required (if more machines are in the VPN). We don't use broadcasting for example.

We use zerotier as VPN.

Reply:
Thanks!